src/docs/sphinx/developerGuide/Contributing/GitWorkflow.rst

```diff
@@ -167,69 +162,30 @@ The main difference with a ``bugfix`` branch is that the primary target branch i
 ``1.2.1``, the merged commit would be tagged with ``1.2.2``.
+Finally, at some point prior to the next minor release, the ``release`` branch
+should be merged back into ``develop`` to incorperate any hotfix changes into 
+``develop``.
 
-An example lifecycle diagram for release/hotfix branchs:
-
-.. code-block:: sh
 
-   A----B----C----D----E----F          (develop)
-         \            /|
-          \          / |
-           \        /  | EAA---EAB     (hotfix)
-            \      /   | /      \
-             \    /    |/        \
-            BA---BB    EA         EB   (release/1.2.x)
-                       (1.2.0)    (1.2.1)
-
-
-In addition to merging into the ``release`` branch, the ``hotfix`` may also target 
-``develop`` if it is appropriate.
-If the bug exists in ``develop`` and there are no plans to fix the bug prior 
-to the next ``major.minor`` release cycle, then the ``hotfix`` should be merged
-into ``develop``.
-As ``develop`` may have diverged significantly from the target ``release`` branch,
-the ``hotfix`` branch may need further revisions for successful integration into
-``develop``. 
-As such, the procedure for integration of a hotfix would be to create a new ``bugfix`` 
-branch that "cherry-picks" the ``hotfix`` commit/s used to patch ``release``, and 
-apply appropriate revisions for a successful merge into ``develop``.
-For example if a ``hotfix`` was contained into commit ``BBB``, then the following
-commands would create the appropriate ``bugfix`` branch:
+An example lifecycle diagram for hotfix branchs:
 
 .. code-block:: sh
 
-  git checkout develop
-  git checkout -b bugfix/neo/freeYourMind
-  git cherry-pick BBB
-  git push -u origin bugfix/neo/freeYourMind
-
-Additional commits to this ``bugfix`` branch may then be added prior to merging 
-into ``develop``.
-
-An example lifecycle diagram for release/hotfix/bugfix branchs:
-
-.. code-block:: sh
-             
-   A----B-----C-----D-----E-------F---G---H--  (develop)
-         \               /|        \     /
-          \             / |        FA---FB     (bugfix/neo/freeYourMind)
-           \           /  |        /
-            \         /   | EAA---EAB          (hotfix)
-             \       /    | /      \
-              \     /     |/        \
-              BA---BB     EA         EB        (release/1.2.x)
-                          (1.2.0)    (1.2.1)
-
-
-Some useful links for use of ``git cherry-pick`` are given here:
-
-`Documentation for git cherry-pick <https://git-scm.com/docs/git-cherry-pick>`_
+                        (tag v1.2.1)  (tag v1.3.0)
+   A----B'---...-----G---------H'-----------J'     (release)
+        ^            ^\       / \           ^
+        |            | \     /   \          |
+        |            |  GA--GB    \         |     (hotfix)
+        |            |             \        |
+   A----B----...-----G--------------H---I---J     (develop)
+                (tag v1.2.0)
```

Comment:
Should not the `hotfix` branch go to both `release` and `develop` (starting from `GB`), while there should be no direct link from `release` to `develop` (`H'` to `H`)

Reply:
I have modified the diagram to show a proper merge-back.